packages/server/controllers/posts.js

```diff
@@ -27,12 +27,14 @@ module.exports = {
     if (body) {
       changeObject.body = body;
     }
-
     changeObject.isDraft = isDraft;
+
     try {
-      const post = await Post.findOneAndUpdate({ _id: id }, changeObject, {
-        new: true,
-      });
+      const post = await Post.findById({ _id: id });
+      for (const key in changeObject) {
+        post[key] = changeObject[key];
+      }
+      await post.save();
```

Comment:
Has this return value been tested? I don't want to send down the old post.

Reply:
Yup! we console logged the post after the loop to make sure and tested the edits but I'll double check.